posthog/management/commands/fix_person_distinct_ids_after_delete.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+
+import structlog
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from posthog.client import sync_execute
+from posthog.models.person import PersonDistinctId
+from posthog.models.person.util import create_person_distinct_id
+
+logger = structlog.get_logger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Fix state for person distinct IDs in ClickHouse after person deletion and id re-use for a single team"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--team-id", default=None, type=int, help="Specify a team to fix data for.")
+        parser.add_argument("--new-version", default=2500, type=int, help="New version value to use")
+        parser.add_argument("--live-run", action="store_true", help="Run changes, default is dry-run")
+
+    def handle(self, *args, **options):
+        run(options)
+
+
```

Comment:
would love some of the context from the PR description to make it here as a comment